src/Recipe.js

```diff
@@ -0,0 +1,14 @@
+class Recipe {
+  constructor(id, image, ingredients, instructions, name, tags) {
+    this.id = id;
+    this.image = image;
+    this.ingredients = ingredients;
+    this.instructions = instructions;
+    this.name = name;
+    this.tags = tags;
+  }
+}
+
+if (typeof module !== 'undefined') {
+  module.exports = Recipe;
+}
```

Comment:
This looks solid, that's a hefty constructor. It's gonna get pretty beefy with the methods as well.